app/lib/ddp.js

```diff
@@ -290,9 +296,12 @@ export default class Socket extends EventEmitter {
 				id,
 				name,
 				params,
-				unsubscribe: () => this.unsubscribe(id)
+				unsubscribe: () => {
+					// Also remove the events attached to the subscription of the collection
+					this.removeAllForEvent(name);
```

Comment:
you should call this method `removeAllListeners` inside of `unsubscribe`

Reply:
to use "removeAllListeners" there then we need to pass the name as an extra parameter on unsubscribe. Otherwise there's no way to link them together.